templates/portfolio/Tutorial/styles.css

```diff
@@ -0,0 +1,105 @@
+
+.portfolio-header {
+  position: relative;
+  background-color: #D8D8D8;
+  background-image: url(images/header-bg.jpg);
+}
+
+.mdl-layout__header-row {
```

Comment:
It's somewhat dangerous to override something on an MDL class directly, since you might later decide to use it elsewhere without the override. Since you already have a `.portfolio-header` class, I'd suggest you change this selector to `.portfolio-header .mdl-layout__header-row`. That way, you're only targeting header rows inside of a `.portfolio-header` and are less likely to surprise yourself in the future.